buildSrc/src/main/java/com/osacky/flank/gradle/FlankGradleExtension.kt

```diff
@@ -6,33 +6,35 @@ import org.gradle.api.NamedDomainObjectContainer
 import org.gradle.api.file.RegularFileProperty
 import org.gradle.api.model.ObjectFactory
 import org.gradle.api.provider.ListProperty
+import org.gradle.api.provider.MapProperty
 import org.gradle.api.provider.Property
 import org.gradle.kotlin.dsl.listProperty
+import org.gradle.kotlin.dsl.mapProperty
 import org.gradle.kotlin.dsl.property
 
 open class FlankGradleExtension @Inject constructor(objects: ObjectFactory) : FladleConfig {
   val flankCoordinates: Property<String> = objects.property(String::class.java).convention("com.github.flank:flank")
   val flankVersion: Property<String> = objects.property(String::class.java).convention("20.05.2")
   // Project id is automatically discovered by default. Use this to override the project id.
-  override var projectId: String? = null
+  override var projectId: Property<String> = objects.property()
   override val serviceAccountCredentials: RegularFileProperty = objects.fileProperty()
-  override var useOrchestrator: Boolean = false
-  override var autoGoogleLogin: Boolean = false
-  override var devices: List<Map<String, String>> = listOf(mapOf("model" to "NexusLowRes", "version" to "28"))
+  override var useOrchestrator: Property<Boolean> = objects.property()
+  override var autoGoogleLogin: Property<Boolean> = objects.property()
+  override var devices: ListProperty<Map<String, String>> = objects.listProperty()
```

Comment:
can we specify the default here using convention? this ensure that no matter where it is accessed, the same default value is used.
```suggestion
  override var devices: ListProperty<Map<String, String>> = objects.listProperty().convention(listOf(mapOf("model" to "NexusLowRes", "version" to "28"))
```